sdk/servicebus/Azure.Messaging.ServiceBus/samples/README.md

```diff
@@ -26,3 +26,5 @@ description: Samples for the Azure.Messaging.ServiceBus client library
 - [Managing rules](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicebus/Azure.Messaging.ServiceBus/samples/Sample12_ManagingRules.md)
 - [Advanced configuration](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicebus/Azure.Messaging.ServiceBus/samples/Sample13_AdvancedConfiguration.md)
 - [Interact with the AMQP message](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicebus/Azure.Messaging.ServiceBus/samples/Sample14_AMQPMessage.md)
+- [Mocking Client Types](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicebus/Azure.Messaging.ServiceBus/samples/Sample15_MockingClientTypes.md)
+- [Cross-receiver message settlement]
```

Comment:
Looks like you missed the link here.

Reply:
Will need to add this after the PR merges.

Reply:
You're probably going to have to strip the brackets.  I think the analyze pass will fail due to being a malformed link.